testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/contextdata/TestRemote.java

```diff
@@ -26,6 +26,6 @@
 @Remote
 public interface TestRemote {
 
-  String invoke();
+    UseCaseValidator invoke(UseCaseValidator useCaseValidator) throws TestException;
```

Comment:
`invoke()` is used by other test class and should be kept.

https://github.com/wildfly/wildfly/blob/3b0cdbe49beb445f8b93f972b4a6b8a4887d1f15/testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ejb/remote/contextdata/ClientInterceptorReturnDataEjbOverHttpTestCase.java#L110